File: largest_palindrome/largest_palindrome_spec.rb
require_relative ('largest_palindrome')


describe '#is_palindrome?' do
  it('returns true for palindromes') do
    expect(is_palindrome?(101)).to be true
  end

  it('returns false for non-palindromes') do
    expect(is_palindrome?(900)).to be false
  end
end

describe '#largest_palindrome' do
  it('returns 906609') do
    expect(largest_palindrome).to eq(906609)
  end
end

File: largest_palindrome/largest_palindrome_spec.js
describe("isPalindrome", function() {
  it('returns true if input is a palindrome', function(){
    expect(isPalindrome(1001)).toBe(true);
  });

  it('returns false if input is not a palindrome', function(){
    expect(isPalindrome(988888)).toBe(false);
  });

});

describe('largestPalindrome', function(){
  it('returns the largest palindrome that is the product of two three digit numbers', function(){
    expect(largestPalindrome()).toBe(906609)
  })

});

